clinica/pydra/utils.py

```diff
@@ -0,0 +1,70 @@
+"""This module contains utility functions which are used across Pydra workflows."""
+import typing as ty
+
+
+def sanitize_fwhm(
+    fwhm: ty.Union[float, ty.Tuple[float], ty.List[float], ty.List[ty.List[float]]],
+) -> ty.List[ty.List[float]]:
```

Comment:
Same as above, makes it clear the output is always a list of triple of floats.

```suggestion
) -> ty.List[ty.Tuple[float, float, float]]:
```